app/destination/page.js

```diff
@@ -1,54 +1,102 @@
 "use client";
 
-import { useState } from 'react';
+import { useState } from "react";
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
 
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
+const PlanetCard = ({
   name,
-  onRemove,
+  description,
   thumbnail,
+  isSelected,
+  index,
+  onAddOrRemovePlanet,
 }) => {
   return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
+    <div className={styles.planetCard}>
+      <img
+        className={styles.planetThumbnail}
+        src={thumbnail}
+        alt={`image of ${name}`}
+      />
+      <div className={styles.planetDescription}>
+        <h2>
+          {name}
+          {isSelected ? "- SELECTED" : ""}
+        </h2>
+        <p>{description}</p>
+      </div>
+      <button
+        className="roundButton"
+        onClick={() => onAddOrRemovePlanet(name, index)}
```

Comment:
```suggestion
        onClick={() => onClick(name)}
```
The card should not be aware that it is used in a list so it can be used in other situations.